docs/community.md

```diff
@@ -1,5 +1,19 @@
-Want to reach the ORAS community and developers?
-We're very interested in feedback and contributions for other artifacts.
+Welcome to the ORAS Project community.
+
+- [Join us](https://slack.cncf.io/) at [CNCF Slack](https://cloud-native.slack.com) under the **#oras** channel
+- Join the weekly community meeting at 4:00 PM PST at [Zoom meeting room](https://zoom.us/j/8079328631?pwd=UzN0NUlmbmcvTTN5L0o5VUQ2YndyQT09) with ID 807 932 8631. See [meeting notes](https://hackmd.io/P-O6n222TcSMoJgHmTTduw?view) for more details
+- Follow us on Twitter: [@orasproject](https://twitter.com/orasproject)
+- Subscribe us on [YouTube](https://www.youtube.com/@cncforasproject) 
+
+## Developing and contributing to ORAS
+
+If you are interested in developing or contributing to ORAS, you can check out the [CLI developer guide](https://oras.land/cli/5_developer_guide/).
```

Comment:
It might be good to have an organization developer guide that can be applicable to any ORAS project. The document referenced here is focused heavily toward the CLI project instead of a generic ORAS development and contribution guide

Reply:
Changed to organization developer guide https://oras.land/contributing/